compiler/tools/fpp-to-cpp/test/array/S1SerializableAc.ref.hpp

```diff
@@ -0,0 +1,302 @@
+// ======================================================================
+// \title  S1SerializableAc.hpp
+// \author Generated by fpp-to-cpp
+// \brief  hpp file for S1 struct
+// ======================================================================
+
+#ifndef M_S1SerializableAc_HPP
+#define M_S1SerializableAc_HPP
+
+#include "Fw/Types/BasicTypes.hpp"
+#include "Fw/Types/Serializable.hpp"
+#include "Fw/Types/String.hpp"
+
+namespace M {
+
+  class S1 :
+    public Fw::Serializable
+  {
+
+    public:
+
+      // ----------------------------------------------------------------------
+      // StringSize80 class
+      // ----------------------------------------------------------------------
+
+      class StringSize80 :
+        public Fw::StringBase
+      {
+
+        public:
+
+          enum {
+            SERIALIZED_SIZE = 80 + sizeof(FwBuffSizeType) //!< Size of buffer + storage of two size words
```

Comment:
I just notice this: Why does it say "Size of buffer + two size words"? Shouldn't it be "Size of string buffer plus size of string length"?

Reply:
Yes that would be clearer -- I think the "two words" was because the type of FwBuffSizeType is U16.

Reply:
OK, let's make a separate issue on this.